support-frontend/assets/pages/new-contributions-landing/components/ContributionFormFields.jsx

```diff
@@ -138,7 +133,7 @@ function FormFields(props: PropTypes) {
             formHasBeenSubmitted={checkoutFormHasBeenSubmitted}
             errorMessage="Please provide your first name"
             required
-            showRequiredLabel={showRequiredLabel}
+            showRequiredLabel
```

Comment:
maybe we could put the required labels in the server-rendered template? since they're not subject to A/B test changes